src/modules/drawing/components/DrawingToolbox.vue

```diff
@@ -93,6 +104,9 @@ export default {
         }
     },
     methods: {
+        showClearConfirmation: function () {
+            this.$refs['clearConfirmation'].show = true
+        },
```

Comment:
Wouldn't it be better (and more understandable) to use a `v-show` or `v-if` on the template itself instead of changing that here in a method?

Reply:
We should open the modal window on button click, so we will need a method to handle click events anyway. Also, in the case of use `v-show` or `v-if`, we will need an additional variable